### PR TITLE
supervisor-pwned

### PR DESCRIPTION
**Pull request by pvizeli**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/more-info/pwned-passwords.markdown [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Pwned Passwords"
+description: "More information on detected PWNED secrets in Home Assistant."
+---
+
+We using following serverce for detecting leaked secrets: https://haveibeenpwned.com/Passwords
```
Comment (fabaff) [inline source/more-info/pwned-passwords.markdown]:
```suggestion
We are using the [Have I Been Pwned (HIBP)](https://haveibeenpwned.com/Passwords) service for detecting leaked secrets.
```

File: source/more-info/pwned-passwords.markdown [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+If you getting a pwned warning, it means you using insecure secrets on that configuration.
+You should imidently upate this secrets.
```
Comment (fabaff) [inline source/more-info/pwned-passwords.markdown]:
```suggestion
You should immediately update this secrets.
```

File: source/more-info/pwned-passwords.markdown [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Pwned Passwords"
+description: "More information on detected PWNED secrets in Home Assistant."
+---
+
+We using following serverce for detecting leaked secrets: https://haveibeenpwned.com/Passwords
+
+If you getting a pwned warning, it means you using insecure secrets on that configuration.
```
Comment (fabaff) [inline source/more-info/pwned-passwords.markdown]:
```suggestion
If you are getting a warning, it means that you are using secrets in your configuration which have been leaked and are publicly known.
```

File: source/more-info/pwned-passwords.markdown [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Pwned Passwords"
+description: "More information on detected PWNED secrets in Home Assistant."
```
Comment (fabaff) [inline source/more-info/pwned-passwords.markdown]:
```suggestion
description: "More information on detected pwned secrets in Home Assistant."
```

File: source/more-info/pwned-passwords.markdown [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Pwned Passwords"
```
Comment (fabaff) [inline source/more-info/pwned-passwords.markdown]:
```suggestion
title: "Pwned passwords and secrets"
```

**Review by fabaff — CHANGES_REQUESTED** [review]
It should also contain a section where is mentioned that the first 5 digits of local data's SHA1 hashes is send to a third-party service.

File: source/more-info/pwned-passwords.markdown [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+title: "Pwned passwords and secrets"
+description: "More information on detected pwned secrets in Home Assistant."
+---
+
+We are using the [Have I Been Pwned (HIBP)](https://haveibeenpwned.com/Passwords) service for detecting leaked secrets.
+
+If you are getting a warning, it means that you are using secrets in your configuration which have been leaked and are publicly known.
+You should immediately update this secrets.
+
+We use [K-Anonymity](https://blog.cloudflare.com/validating-leaked-passwords-with-k-anonymity/) to send secure and anonyme 5 characters from a 40 characters hashed password.
```
Comment (frenck) [inline source/more-info/pwned-passwords.markdown]: How about?: 

```suggestion
We are using the [Have I Been Pwned (HIBP)](https://haveibeenpwned.com/Passwords) service for detecting leaked or compromised secrets, like passwords.

If you get a warning about it, it means that you are using secrets in your configuration which have been leaked and are publicly known. It is strongly advised to change these secrets by a more secure alternative as soon as possible.

Please note; this feature does not send out your secrets to check this. Your secrets and privacy is guaranteed by a [K-Anonymity][k-anonymity]. Your secrets are hashed, the first 5 characters of the hash result are used to query Have I Been Pwned. Have I Been Pwned returns the results of possible password hashes that match, we check the last part of the password hash against this list locally.

[Read more about K-Anonymity on this CloudFlare blog post][k-anonymity].

[k-anonymity]: https://blog.cloudflare.com/validating-leaked-passwords-with-k-anonymity/
```

**Review by frenck — APPROVED** [review]

**Pull request merged**